test/ArgumentForwarding.Tests/ArgumentForwarding.Tests.csproj

```diff
@@ -10,22 +10,27 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\Cli\dotnet\dotnet.csproj" />
     <ProjectReference Include="..\Microsoft.NET.TestFramework\Microsoft.NET.TestFramework.csproj" />
+    <ProjectReference Include="..\ArgumentsReflector\ArgumentsReflector.csproj" ReferenceOutputAssembly="false" OutputItemType="ArgumentsReflectorOutput" />
```

Comment:
Consider setting the `Targets` metadata to `Publish` - this'll overwrite the default of `Build` and you won't have to modify `ArgumentsReflector.csproj` at all.